tests/common.bash

```diff
@@ -22,103 +69,65 @@ function kubectl_apply_should_fail_with_message {
 	kubectl_delete_by_type_and_name configmap $1
 }
 
-function wait_for_all_cluster_admission_policies_to_be_active {
-	wait_for_all_cluster_admission_policy_condition PolicyActive
+function wait_for_admission_policy {
+	kubectl wait --timeout $TIMEOUT --for=condition="$1" admissionpolicies --all -A
 }
 
-function wait_for_all_cluster_admission_policy_condition {
-	run kubectl --context $CLUSTER_CONTEXT wait --timeout $TIMEOUT --for=condition="$1" clusteradmissionpolicies --all
-	[ "$status" -eq 0 ]
-}
-
-function wait_for_all_admission_policies_to_be_active {
-	run kubectl --context $CLUSTER_CONTEXT wait --timeout $TIMEOUT --for=condition=PolicyActive admissionpolicies -A --all
-	[ "$status" -eq 0 ]
-}
-
-function wait_for_all_admission_policy_condition {
-	run kubectl --context $CLUSTER_CONTEXT wait --timeout $TIMEOUT --for=condition="$1" admissionpolicies -A --all
-	[ "$status" -eq 0 ]
-}
-
-function wait_for_all_pods_to_be_ready {
-	wait_for_default_policy_server_rollout
-	run kubectl --context $CLUSTER_CONTEXT wait --for=condition=Ready --timeout $TIMEOUT -n $NAMESPACE pod --all
-	[ "$status" -eq 0 ]
+function wait_for_cluster_admission_policy {
+	kubectl wait --timeout $TIMEOUT --for=condition="$1" clusteradmissionpolicies --all
 }
 
 function wait_for_default_policy_server_rollout {
-	wait_for_policy_server_rollout default
+ 	revision=$(kubectl -n $NAMESPACE get "deployment/policy-server-default" -o json | jq -r '.metadata.annotations."deployment.kubernetes.io/revision"')
+ 	wait_rollout -n $NAMESPACE --revision $revision "deployment/policy-server-default"
 }
 
 function default_policy_server_rollout_should_fail {
-	revision=$(kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE get "deployment/policy-server-default" -o json | jq ".metadata.annotations.\"deployment.kubernetes.io/revision\"" | sed "s/\"//g")
-	run kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE rollout status --revision $revision --timeout $TIMEOUT "deployment/policy-server-default"
-	[ "$status" -ne 0 ]
-}
-
-function wait_for_policy_server_rollout {
-	revision=$(kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE get "deployment/policy-server-$1" -o json | jq ".metadata.annotations.\"deployment.kubernetes.io/revision\"" | sed "s/\"//g")
-	run kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE rollout status --revision $revision "deployment/policy-server-$1"
+	revision=$(kubectl -n $NAMESPACE get "deployment/policy-server-default" -o json | jq -r '.metadata.annotations."deployment.kubernetes.io/revision"')
+	run ! kubectl -n $NAMESPACE rollout status --revision $revision --timeout $TIMEOUT "deployment/policy-server-default"
 }
 
 function default_policy_server_should_have_log_line {
-	policy_server_should_have_log_line default "$1"
-}
-
-function policy_server_should_have_log_line {
-	run kubectl --context $CLUSTER_CONTEXT logs -n $NAMESPACE -lapp="kubewarden-policy-server-$1" | grep "$2"
+	kubectl logs -n $NAMESPACE -lapp="kubewarden-policy-server-default" | grep "$1"
```

Comment:
shouldn't this have a `run`?

Reply:
No, this is explained in https://bats-core.readthedocs.io/en/stable/gotchas.html#my-piped-command-does-not-work-under-run
`run` eats all the output and you grep on empty string - which always fails